accepted/0000-sbom-command.md

```diff
@@ -207,51 +207,57 @@ The proposed CycloneDX SBOM generated for the project above would look like the
 
 ```json
 {
-  "$schema": "https://cyclonedx.org/schema/bom-1.5.schema.json",
+  "$schema": "http://cyclonedx.org/schema/bom-1.5.schema.json",
```

Comment:
why would a schema for a security-based format be served over http? O.o

Reply:
I believe this merely functions as an [identifier](https://github.com/CycloneDX/specification/blob/fd4d383658196992364e5d62568a48c431ace515/schema/bom-1.5.schema.json#L3) and is not actually resolvable at this URL.

Reply:
> https://github.com/npm/rfcs/pull/714#discussion_r1323681082
exactly

this string is an identifier, like a name. 

it is not expected or treated as a URL. it just is common practice to
- use a URL-like prefix to indicate ownership
- use a URL that is actually resolvable, so people have an easy way to fetch the related schema document.

Reply:
thanks, that makes sense, but it's very strange to me to use the http form as an identifier rather than the https form